README.md

```diff
@@ -44,8 +44,16 @@ Update the `settings.local.json` file with the desired input to the tool as desi
 `tagcompare` is the main entry point for the tool, it will capture and compare tags 
 with the given input in your `settings.local.json`
 
-All outputs generated by the tool will be stored in `/tmp/tagcompare`
+### Commandline arguments:
+Commandline arguments can optionally be used to configure the tagcompare run.
+`tagcompare --help` will show the current list of supported options.
+
+Note:Options passed through the commandline will override the `settings.local.json`
+
+### Outputs:
+Outputs generated by the tool will be stored in `/~/tagcompare`
 
 ## Running tests
 `make test` or `tox` will run unit tests
 `make test-all` will run all the tests
+`./test <path_to_test>` will run a specific test
```

Comment:
If only a file path is passed in this will run a "set" of tests rather than a specific test, ex
 tagcompare/test/test_settings.py  will run 
- test_domain(),
- test_campagigns(), 
- test_publishers() 
- ...